src/js/components/Exchange/AccountBalanceLayout.js

```diff
@@ -60,10 +60,10 @@ const AccountBalanceLayout = (props) => {
         // }
         break;
       case "Bal":
-        res = Object.keys(tokens).map(key => tokens[key]).sort((a, b) => {return (props.sortValue ? -1 : 1)*(converts.subOfTwoNumber(a.balance, b.balance))} )
+        res = Object.keys(tokens).map(key => tokens[key]).sort((a, b) => {return (props.sortValue ? -1 : 1)*(converts.subOfTwoNumber(converts.toT(a.balance, a.decimals), converts.toT(b.balance, b.decimals)))} )
         break;
       case "USDT":
-        res = Object.keys(tokens).map(key => tokens[key]).sort((a, b) => {return (props.sortValue ? -1 : 1)*(converts.subOfTwoNumber(converts.multiplyOfTwoNumber(a.balance, a.rateUSD), converts.multiplyOfTwoNumber(b.balance, b.rateUSD)))} )
+        res = Object.keys(tokens).map(key => tokens[key]).sort((a, b) => {return (props.sortValue ? -1 : 1)*(converts.subOfTwoNumber(converts.multiplyOfTwoNumber(converts.toT(a.balance, a.decimals), a.rateUSD), converts.multiplyOfTwoNumber(converts.toT(b.balance, b.decimals), b.rateUSD)))} )
```

Comment:
It should break line here, its hard for us to check logic with very long line

Reply:
Updated